### ISSUE TITLE
null additional_data parsed from notification

### ISSUE THREAD

**Issue by unieagle** [killbill/killbill-adyen-plugin#65]
https://github.com/killbill/killbill-adyen-plugin/blob/e4fcf8e50f0c0a7c2ad8a5b212ad4a40ed58a806/src/main/java/org/killbill/billing/plugin/adyen/client/model/NotificationItem.java#L46

Hi @pierre @sbrossie , do you know why we initially ignored the addtional_data? for some chargebacks, adyen put the real reason code only in this field.

**Comment by pierre** [comment]
Indeed, it looks like an (old) oversight. Do you have some bandwidth to work on a PR?
